std_msgs/Header header
geometry_msgs/Pose pose_error
float64[7] tau_pos
float64[7] tau_or
float64[7] tau_internal
float64[7] tau_fric
float64[36] cartesian_stiffness
float64[7] tau_null